api/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Python version 
+FROM python:3.7
```

Comment:
Hey @vanessavirgo94 - Just checking if the version should be 3.8 as thats what we been using at Peak ?

Reply:
Leave this for now